sysbox-in-docker/Dockerfile.debian-bullseye

```diff
@@ -0,0 +1,59 @@
+#
+# Sysbox-In-Docker Container Dockerfile (Debian-Bullseye image)
+#
+# This Dockerfile creates the sysbox-in-docker container image, which holds
+# all Sysbox binaries and its dependencies. The goal is to allow users to run
+# an entire Sysbox sandbox within a container.
+#
+# Notice that the image must be run as a privileged container, and a few resources
+# must be bind-mounted to meet Sysbox requirements as well as those of system-level
+# apps running in inner containers. See instructions below.
+#
+# Instructions:
+#
+# * Image creation:
+#
+#   $ make sysbox-in-docker debian-bullseye
+#
+# * Container creation:
+#
+# docker run -it --privileged --rm --hostname sysbox-in-docker --name sysbox-in-docker \
+#                -v /var/tmp/sysbox-test-var-lib-docker:/var/lib/docker \
```

Comment:
Remove reference to "sysbox-test-var-lib-docker" (it's a sysbox test dir). Use something like `-v some-vol:/var/lib/docker`

Reply:
Done.